configure.ac

```diff
@@ -262,6 +262,14 @@ AC_ARG_WITH(system-install-dir,
 SYSTEM_INSTALL_DIR=$with_system_install_dir
 AC_SUBST(SYSTEM_INSTALL_DIR)
 
+AC_ARG_WITH(external-install-dir,
+           [AS_HELP_STRING([--with-external-install-dir=DIR],
+                           [Location of the external installation [LOCALSTATEDIR/endless-extra/flatpak]])],
+           [],
+           [with_external_install_dir='$(localstatedir)/endless-extra/flatpak'])
+EXTERNAL_INSTALL_DIR=$with_external_install_dir
+AC_SUBST(EXTERNAL_INSTALL_DIR)
```

Comment:
I think you're absolutely right. TBH, I've never consider this as an upstreameable thing due to its "short-term fix" nature, but it could actually be interesting upstream as an intermediate step too, so I'll keep that in mind. Thanks